lib/solaar/ui/config_panel.py

```diff
@@ -76,15 +76,37 @@ def _combo_notify(cbbox, s):
 	c.connect('changed', _combo_notify, setting)
 	return c
 
-# def _create_slider_control(setting):
-# 	def _slider_notify(slider, s):
-# 		if slider.get_sensitive():
-# 			_apply_queue.put(('write', s, slider.get_value(), slider.get_parent()))
-#
-# 	c = Gtk.Scale(setting.choices)
-# 	c.connect('value-changed', _slider_notify, setting)
-#
-# 	return c
+def _create_slider_control(setting):
+	class SliderControl:
+		__slots__ = ('gtk_range', 'timer', 'setting')
+		def __init__(self, setting):
+			self.setting = setting
+			self.timer = None
+
+			self.gtk_range = Gtk.Scale()
+			self.gtk_range.set_range(*self.setting.range)
+			self.gtk_range.set_round_digits(0)
+			self.gtk_range.set_digits(0)
+			self.gtk_range.set_increments(1, 5)
+			self.gtk_range.connect('value-changed',
+			                       lambda _, c: c._changed(),
+								   self)
+
+		def _write(self):
+			_write_async(self.setting,
+			             int(self.gtk_range.get_value()),
+						 self.gtk_range.get_parent())
+			self.timer.cancel()
+
+		def _changed(self):
+			if self.gtk_range.get_sensitive():
+				if self.timer:
+					self.timer.cancel()
+				self.timer = _Timer(0.5, lambda: GLib.idle_add(self._write))
```

Comment:
This could give issues when the user quickly closes the dialog, but I guess that is acceptable.